MobileWallet/Common/Extensions/LAContext.swift

```diff
@@ -48,6 +48,18 @@ extension LAContext {
         case pin
     }
 
+    enum AuthenticateUserReason: String {
+        case logIn = "Log in to your account"
+        case userVerification = "User Verification"
+
+        public func localized(args: CVarArg...) -> String {
+            let localizedString = NSLocalizedString(self.rawValue, comment: "")
```

Comment:
This won't work when exporting the language files. The actual string needs to be inside this method for the exporting to work. Rather do this:

`case logIn = NSLocalizedString("Log in to your account", "Authentication")`